app/domain/errors.rb

```diff
@@ -19,6 +19,12 @@ module Conjur
         code: "CONJ00037E"
       )
 
+      PasswordComplexityBreak = Util::TrackableErrorClass.new(
```

Comment:
Could we change the name of this error type to something like:

```suggestion
      InsufficientPasswordComplexity = Util::TrackableErrorClass.new(
```

Or even just `PasswordComplexityError`. The `Break` part is a little odd.